Library/Formula/parse-cli.rb

```diff
@@ -0,0 +1,17 @@
+class ParseCli < Formula
+  homepage "https://parse.com"
+  url "https://www.parse.com/downloads/cloud_code/parse",
```

Comment:
This looks alarmingly like a dynamic rather than static link. Is it a link that is guaranteed to not suddenly change versions and checksums underneath us?